.license

```diff
@@ -0,0 +1,4 @@
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
```

Comment:
This is the wrong text for our MIT license.
Please see https://github.com/spryker/code-sniffer/blob/master/.license